boards/arm/stm32f3_seco_d23/stm32f3_seco_d23_defconfig

```diff
@@ -0,0 +1,38 @@
+# SPDX-License-Identifier: Apache-2.0
+# SECO SBC-D23 board defconfig
+#
+# Copyright (c) 2022, SECO Spa
+
+CONFIG_SOC_SERIES_STM32F3X=y
+CONFIG_SOC_STM32F303XC=y
+
+# 72MHz system clock
+CONFIG_SYS_CLOCK_HW_CYCLES_PER_SEC=72000000
+
+# Floating Point Options
+CONFIG_FPU=y
```

Comment:
Not sure this belongs to board configuration.

Reply:
Removing